assets/airports/cyhz.json

```diff
@@ -0,0 +1,2633 @@
+                ["N44d26m15.00", "W063d42m18.00"],
+                ["N44d26m44.00", "W063d43m10.00"],
+                ["N44d27m14.00", "W063d44m00.00"],
+                ["N44d27m47.00", "W063d44m48.00"],
+                ["N44d28m21.00", "W063d45m33.00"],
+                ["N44d28m57.00", "W063d46m16.00"],
+                ["N44d29m34.00", "W063d46m56.00"],
+                ["N44d30m13.00", "W063d47m34.00"],
+                ["N44d30m53.00", "W063d48m08.00"],
+                ["N44d31m34.00", "W063d48m40.00"],
+                ["N44d32m16.00", "W063d49m08.00"],
+                ["N44d33m00.00", "W063d49m33.00"],
+                ["N44d33m44.00", "W063d49m56.00"],
+                ["N44d34m29.00", "W063d50m14.00"],
+                ["N44d35m15.00", "W063d50m30.00"],
+                ["N44d36m01.00", "W063d50m42.00"],
+                ["N44d36m47.00", "W063d50m51.00"],
+                ["N44d37m34.00", "W063d50m57.00"],
+                ["N44d38m21.00", "W063d50m59.00"],
+                ["N44d39m08.00", "W063d50m57.00"]
+            ]
+        }
+    ],
+    "fixes": {
+        "AGMIR": ["N45d13.70", "W061d57.33"],
+        "ALLEX": ["N44d25.00", "W067d00.00"],
+        "AMUNA": ["N44d57.74", "W063d07.54"],
+        "ARDEE": ["N45d07.40", "W064d23.17"],
+        "AVIGU": ["N44d45.72", "W063d29.19"],
+        "AVRIX": ["N44d57.30", "W063d21.37"],
+        "BAFDY": ["N44d46.66", "W064d46.95"],
+        "BIBEP": ["N44d42.81", "W063d32.06"],
+        "BOSUB": ["N45d11.83", "W062d55.52"],
+        "CAAPE": ["N45d18.00", "W065d17.80"],
+        "CADIL": ["N47d44.12", "W060d25.47"],
+        "CAYLY": ["N45d37.27", "W062d14.50"],
+        "DUNUB": ["N44d58.18", "W063d52.51"],
+        "DUSEN": ["N44d59.84", "W064d11.79"],
+        "DUTIR": ["N45d04.68", "W063d28.50"],
+        "DUTSA": ["N44d50.48", "W063d24.80"],
+        "DUXIK": ["N44d49.02", "W063d21.85"],
+        "EBKID": ["N44d53.42", "W065d30.08"],
+        "EBLIT": ["N44d40.17", "W063d16.42"],
+        "EBLUV": ["N44d47.75", "W063d31.52"],
+        "EBONY": ["N44d54.14", "W067d09.39"],
+        "ELERI": ["N42d34.47", "W064d23.41"],
+        "EMBAX": ["N44d43.06", "W063d22.17"],
+        "ETGAR": ["N44d25.16", "W064d09.78"],
+        "ETMEP": ["N44d49.70", "W063d41.29"],
+        "GABAP": ["N44d51.24", "W063d14.09"],
+        "GOLKE": ["N44d46.84", "W063d25.05"],
+        "GOSUG": ["N44d53.81", "W063d42.06"],
+        "HIDIG": ["N44d40.84", "W064d00.97"],
+        "HOROW": ["N44d46.54", "W063d45.51"],
+        "IGTAS": ["N45d04.97", "W062d59.38"],
+        "IGVOP": ["N45d03.52", "W063d38.66"],
+        "ILOTA": ["N44d52.52", "W063d48.90"],
+        "IMANO": ["N44d55.98", "W063d35.78"],
+        "IMESA": ["N44d56.14", "W063d39.47"],
+        "KEBNI": ["N45d07.89", "W064d11.37"],
+        "KEBOT": ["N44d59.18", "W063d22.63"],
+        "KESMO": ["N44d54.01", "W063d20.99"],
+        "KIXIT": ["N44d48.33", "W063d08.31"],
+        "LEROS": ["N45d01.90", "W063d22.22"],
```

Comment:
The problems we were having with the ILS's not lining up weren't due to inaccuracies with the runway coordinates, but rather that the coordinates for `LEROS` are slightly off